pkg/fixtures/example_project/fiz/fiz.go

```diff
@@ -0,0 +1,11 @@
+package fiz
+
+import (
+	"github.com/vektra/mockery/v2/pkg/fixtures/example_project/fiz/fizi/fiz1"
+	"github.com/vektra/mockery/v2/pkg/fixtures/example_project/fiz/fizi/fiz2"
+)
+
+type Fiz interface {
+	FA(f fiz1.Fiz1)
+	FB(f fiz2.Fiz2)
+}
```

Comment:
I would request you add a short README.md to `example_project/fiz` explaining what this package is testing.

Also I'd like it to be renamed to something more descriptive like `replaceType` or something.

Reply:
renamed the files and types, and added a README.